terraform/modules/front_door/load_balancer_access_log_bucket.tf

```diff
@@ -20,7 +20,7 @@ resource "aws_s3_bucket_public_access_block" "load_balancer_access_logs" {
 
 data "aws_elb_service_account" "current" {}
 
-resource "aws_s3_bucket_policy" "force_ssl_and_allow_access_logs" {
+resource "aws_s3_bucket_policy" "force_ssl_and_allow_load_balancer_access_logs" {

```

Comment:
I'm anticipating a potential clash with an upcoming Cloudfront access logging PR, so have renamed this file and this bucket policy to avoid it (since there will be two sets of similar resources in the front door module, one for cloudfront and one for load balancer access logging)